board/static/vendors/bootstrap-progressbar/css/bootstrap-progressbar-2.0.0.css

```diff
@@ -0,0 +1,310 @@
+/*! bootstrap-progressbar v0.9.0 | Copyright (c) 2012-2015 Stephan Groß | MIT license | http://www.minddust.com */
```

Comment:
For all these different versions of bootstrap-progressbar-x.y.z.css, do we need all of these?

Reply:
fixed